docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM flink:1.15.2
+
+ENV HADOOP_VERSION=3.2.4
+
+#Prepare hadoop
+WORKDIR /opt
+RUN curl -O https://dlcdn.apache.org/hadoop/common/hadoop-$HADOOP_VERSION/hadoop-$HADOOP_VERSION.tar.gz && \
+    tar -xzvf hadoop-$HADOOP_VERSION.tar.gz
+
+ENV PATH=/opt/hadoop-$HADOOP_VERSION/bin/:$PATH
+
+# Prepare flink
+WORKDIR ${FLINK_HOME}
+RUN mkdir plugins/oss-fs-hadoop && \
+    mkdir plugins/s3-fs-hadoop && \
+    ln -s ../../opt/$(ls opt | grep oss-fs-hadoop) plugins/oss-fs-hadoop/$(ls opt | grep oss-fs-hadoop) && \
```

Comment:
Since the Flink version is fixed to be 1.15.2, would it be simpler to explicitly specify the filename here?